src/library_webgl.js

```diff
@@ -3075,7 +3084,7 @@ var LibraryGL = {
 #endif
     // Remove comments and C-preprocess the input shader first, so that we can appropriately
     // parse the layout location directives.
-    source = preprocess_c_code(remove_cpp_comments_in_shaders(source));
```

Comment:
This is very specific. Leave a comment explaining why this particular predefined variable must be defined here?

Reply:
oops, it was on auto-merge :) np